README.md

```diff
@@ -49,36 +63,166 @@ const { MediaContextProvider, Media, createMediaStyle } = createMedia({
 })
 
 const App = () => (
-  <>
-    <Style>{createMediaStyle()}</Style>
+  <MediaContextProvider>
+    <Media at='sm'>
+      <MobileApp />
+    </Media>
+    <Media at='md'>
+      <TabletApp />
+    </Media>
+    <Media greaterThanOrEqual='lg'>
+      <DesktopApp />
+    </Media>
+  </MediaContextProvider>
+)
+
+ReactDOM.render(<App />, document.getElementById("react"))
+```
+
+## Server-side Rendering (SSR) Usage
+
+The first important thing to note is that when server-rendering with
+`@artsy/fresnel`, all breakpoints get rendered by the server. Each Media
+component is wrapped by plain CSS that will only show that breakpoint if it
+matches the user's current browser size. This means that the client can
+accurately start rendering the HTML/CSS while it receives it, which is long
+before the React application has booted. This improves perceived performance for
+end-users.
+
+Why not just the one that the current device needs? A couple reasons. First, we
+can't accurately identify which breakpoint your device needs on the server. We
+could use a library to sniff the browser user-agent, but those aren't always
+accurate, and they wouldn't give us all the information we need to know when we
+are server-rendering.
+
+What this means in practice is we can automatically match layouts for mobile,
```

Comment:
What this means in practice is `that` we can automatically match layouts for mobile ?